src/components/ClientSideLoggingToolMenu/BaseClientSideLoggingToolMenu.tsx

```diff
@@ -70,6 +85,18 @@ function BaseClientSideLoggingToolMenu({shouldStoreLogs, capturedLogs, file, onS
                     onToggle={onToggle}
                 />
             </TestToolRow>
+            {!!shouldStoreLogs && isViaTestToolsModal && (
+                <TestToolRow title={translate('initialSettingsPage.troubleshoot.debugConsole')}>
+                    <Button
+                        small
+                        text={translate('initialSettingsPage.debugConsole.viewConsole')}
+                        onPress={() => {
+                            toggleTestToolsModal()
+                            Navigation.navigate(ROUTES.SETTINGS_CONSOLE.getRoute());
```

Comment:
```suggestion
                            Navigation.navigate(ROUTES.SETTINGS_CONSOLE.getRoute(route));
```

where route is the route you want to come back to (probably `Navigation.getActiveRoute()`, but you'd have to test that)